main/urls.py

```diff
@@ -11,4 +11,7 @@
     path("accounts/", include("django.contrib.auth.urls")),
     path("profile/", views.UserUpdateView.as_view(), name="profile"),
     path("about/", views.AboutPageView.as_view(), name="about"),
+    path("terms/", views.TermsPageView.as_view(), name="terms"),
+    path("contact/", views.ContactPageView.as_view(), name="contact"),
+    path("question/", views.QuestionPageView.as_view(), name="question"),
```

Comment:
Maybe name this one "self-assess"
```suggestion
    path("self-assess/", views.QuestionPageView.as_view(), name="self-assess"),
```
And update in other places accordingly.

Reply:
Done.